supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -74,6 +74,35 @@ If you refer to those tickets without including a link, inform the user. See the
 
 “Some referenced tickets are not linked. This means that the ticket is not accessible without Red Hat credentials."
 
+[[release-note-asciidoc-format]]
+== Release note formatting in AsciiDoc
+To avoid nesting headings excessively, treat each release note as a description list item. This format is also compatible with the DITA/AEM migration. For additional formatting examples, you can inspect the link:https://github.com/redhat-documentation/supplementary-style-guide/blob/main/supplementary_style_guide/style_guidelines/release-notes.adoc?plain=1[Release notes] file in the `supplementary-style-guide` repository.
```

Comment:
```suggestion
To avoid nesting headings excessively, treat each release note as a description list item. This format is also compatible with the DITA/AEM migration. For additional formatting examples, see the link:https://github.com/redhat-documentation/supplementary-style-guide/blob/main/supplementary_style_guide/style_guidelines/release-notes.adoc?plain=1[Release notes] file in the `supplementary-style-guide` repository.
```